src/views/tickets/rsvp.php

```diff
@@ -108,7 +113,7 @@ class="tribe-ticket-quantity"
 		}
 		?>
 
-		<?php if ( $is_there_any_product_to_sell ) : ?>
+		<?php if ( $are_products_available ) : ?>
```

Comment:
Why are we testing a boolean to create a boolean with the same value? This is the only place we use this, but `is_in_stock()` (and thus `$is_there_any_product_to_sell`)  should return a boolean. Feels like this is a band-aid for another issue?

Reply:
There _will_ be an easier way to do this check later (post TER) but this is 👍  for now.